src/components/variable.jsx

```diff
@@ -107,14 +108,44 @@ const Variable = React.createClass({
                 {startDate && stopDate &&
                   <h3>Du <FormattedDate value={startDate} /> au <FormattedDate value={stopDate} />&nbsp;:</h3>
                 }
-                <pre><code>{formulas[date].content}</code></pre>
+                <pre><code>{this.renderLinkedFormulaVariables(formulas[date].content)}</code></pre>
               </div>
             )
           })
         }
       </div>
     )
   },
+  link(substring) {
+    return <Link key={substring + Math.random()} to={substring}>{substring}</Link> 
+  },
+  isVariable(substring) {
+    //Ignore every text that isn't a single word like a variable must be:
+    return (!substring.includes(" ") && this.props.variables[substring])
+  },
+  linkCodeSplits(splits, separator) {
+    var previousIsLink = false
+    return splits.map((substring, index) => {
+      if (this.isVariable(substring)) {
+        substring = this.link(substring)
+        previousIsLink = true
+      } else {
+        substring = previousIsLink ? separator + substring : substring  //No jsx and separator concatenation.
+        substring = index < splits.length - 1 ? substring + separator : substring
+        previousIsLink = false
+      }
+      return substring
+    })
+  },
+  //Change every OpenFisca variable in the formula by a link to the variable page:
+  renderLinkedFormulaVariables(formula) {
+    //Split on double quotes first (preventing collision with Link):
+    var splits = this.linkCodeSplits(formula.split('"'), '"') 
+    for (var s of splits) {
+      s = this.linkCodeSplits(s.split("'"), "'")
+    }
+    return s
```

Comment:
That seems to be only returning the last split... I doubt this works in a variable with several variables calls surrounded by `""`